java/Ice/greeter/client/src/main/java/com/zeroc/greeter/Client.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) ZeroC, Inc.
+
+package com.zeroc.greeter;
+
+import com.zeroc.Ice.Communicator;
+import com.zeroc.Ice.Util;
```

Comment:
If you decide to leave the `Util` call qualified, should also remove this from the `import` list.
In Java, `import`s are just glorified `using`.
```suggestion
```